src/core/router.js

```diff
@@ -46,7 +47,8 @@ var logger = loggerFactory('router');
  * @author Bertrand Chevrier <bertrand@taotesting.com>
  * @exports router
  */
-var router = {
+var router = eventifier({
+    activeController: null,
```

Comment:
Why not use a local variable instead?
And what will happen when several controllers are loaded for the same route?

Reply:
Moved to the local variable.
The case for several controllers is covered